lib/commands/update.js

```diff
@@ -75,7 +76,27 @@ class UpdateCommand extends Command {
             task: this.removeOldVersions
         }];
 
-        return this.ui.run(() => this.version(context), 'Checking for latest Ghost version').then((result) => {
+        return this.ui.run(
```

Comment:
This looks really similar to the migrations task - is it possible to run the migration command, similar to the way [install calls setup](https://github.com/TryGhost/Ghost-CLI/blob/master/lib/commands/install.js#L85)?

Reply:
good idea!